### PR TITLE
fix: Move European map viewport slightly eastwards

### PR DESCRIPTION
**Pull request by Kalli**
Hi there Owid gang!

_tl:dr: Moves European map viewport slightly eastwards so that Cyprus is visible in all screen sizes_

I was playing around with the grapher, kicking the proverbial tires, and figured I might take a stab at a small issue. So this is an attempt at a fix for #1428. 

## The problem 

In narrow viewports (such as on mobile phones) Cyprus was being cropped out of the European map as the viewport was centred too far westwards. The fix was small, nudging the center of the viewport for the European map 3 percentage points eastward. I found that was the happy medium to still have the westernmost part of Europe (Iceland) visible and this came out better than playing around with the projections for Europe. On larger screen sizes the change is barely noticible.

## How to test 

* Load a map chart
* Select Europe from the continent drop down
* Try various screen widths, verify that Cyprus is visible in them all as well as the rest of the continent.
* You can also check out the screenshots below

## Before

<details><summary>Click for mobile before screenshot</summary> 

europe-owid-mobile-map-before

</details>

<details><summary>Click for tablet before screenshot</summary> 

europe-owid-tablet-map-before

</details>

## After

<details><summary>Click for mobile after screenshot</summary> 

europe-owid-mobile-map-after

</details>

<details><summary>Click for tablet after screenshot</summary> 

europe-owid-tablet-map-after

</details>

I hope this was helpful. Let me know if there is anything I should improve here. Otherwise thank you for a great tool and website, I'm a big fan, keep on keepin' on!

### CODE REVIEW

**Comment by marcelgerber** [comment]
Hi Karl, this is great, thank you so much! Very much appreciated!

I tested this and it indeed looks like, no matter the viewport size, Cyprus is now _always_ visible.
Svalbard is slightly cut off sometimes, and Russia of course, but that's fine since they are still visible.

Greetings to beautiful Oslo 👋🏻 And keep on coming with the fixes if you feel like it :)

**Comment by marcelgerber** [comment]
This now went live 🙌🏻

**Comment by Kalli** [comment]
Great, happy to hear it! Thank you Marcel and Our World in Data.
